src/utils/featureFlags.ts

```diff
@@ -1,5 +1,15 @@
+import { ADVANCED_ORDERS_FEATURE_FLAG } from 'constants/featureFlags'
+
+// We can define here some flags to be enabled while we develop
+// TODO: update this before the deployment
+const ENABLED_FOR_DEVELOP = [ADVANCED_ORDERS_FEATURE_FLAG]
```

Comment:
I still think we should use a launch darkly feature flag for this

Reply:
Created issue: https://github.com/cowprotocol/cowswap/issues/2550